packages/transformers/webextension/src/WebExtensionTransformer.js

```diff
@@ -306,50 +307,93 @@ async function collectDependencies(
       }
     }
   } else {
-    if (program.background?.service_worker) {
-      program.background.service_worker = asset.addURLDependency(
-        program.background.service_worker,
-        {
-          bundleBehavior: 'isolated',
-          loc: {
-            filePath,
-            ...getJSONSourceLocation(
-              ptrs['/background/service_worker'],
-              'value',
-            ),
-          },
-          env: {
-            context: 'service-worker',
-            sourceType:
-              program.background.type == 'module' ? 'module' : 'script',
+    if (program.browser_specific_settings?.gecko) {
```

Comment:
📝 Using `browser_specific_settings.gecko` to switch between Firefox and Chrome. Firefox add-on need an ID in their manifest.json when submitted to AMO.

> All Manifest V3 extensions need an add-on ID in their manifest.json when submitted to AMO.
https://extensionworkshop.com/documentation/develop/extensions-and-the-add-on-id/

But, it's not necessary in debugging. I want to know some useful flags.